scripts/run_gce_system_tests.py

```diff
@@ -53,4 +53,4 @@ def test_token_info(self):
 
 
 if __name__ == '__main__':
```

Comment:
Might as well go ahead and remove these clauses since you're here. They're not needed.